analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Branching.cs

```diff
@@ -109,16 +109,20 @@ public void Branching_PersistSymbols_BetweenBlocks()
     Tag(""ElseFirst"", first);
     Tag(""ElseSecond"", second);
 }";
-            var collector = SETestContext.CreateCS(code, new SetTestConstraintCheck()).Collector;
+            var collector = SETestContext.CreateCS(code, new SetTestConstraintCheck()).Validator;
             collector.ValidateTag("IfFirst", x => x.HasConstraint(TestConstraint.First).Should().BeTrue());
             collector.ValidateTag("IfSecond", x => x.HasConstraint(TestConstraint.Second).Should().BeTrue());
             collector.ValidateTag("ElseFirst", x => x.HasConstraint(TestConstraint.First).Should().BeTrue());
             collector.ValidateTag("ElseSecond", x => x.HasConstraint(TestConstraint.Second).Should().BeTrue());
         }
 
         [TestMethod]
-        public void ExitReached_SimpleFlow() =>
-            SETestContext.CreateCS("var a = true;").Collector.ValidateExitReachCount(1);
+        public void ExitReached_SimpleFlow()
```

Comment:
We should update the name after extending the responsibility. Something like `EndNotifications_SimpleFlow()`?

Reply:
Similar for others below